src/record/record-transition.js

```diff
@@ -147,6 +148,11 @@ RecordTransition.prototype.add = function (socketWrapper, version, message) {
       return
     }
 
+    if(!utils.isOfType(data, 'object') && !utils.isOfType(data, 'array')) {
```

Comment:
We try to keep our dependency tree as small as possible. My point is more that we have OBJECT constants in multiple places and since you added a new convenience method it would be nice to move across to using your utility without having to define strings as arguments.

Not fussed about it though!